great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -674,10 +674,8 @@ def get_expectation_suite(
             expectations_schema_dict: dict = cast(
                 dict, self.expectations_store.get(key)
             )
-        except StoreBackendError:
-            raise DataContextError(
-                f"Unable to load Expectation Suite {key.resource_name or key.id}"
-            )
+        except:
+            return None
```

Comment:
What is the case where this should return `None`? Should this just raise whatever error occurred instead?